data_validation/src/spark_rapids_validation_tool/dataproc_wrapper.py

```diff
@@ -79,6 +81,29 @@ def validation(self,
             return
         getattr(validate, check)()
 
+    def validation(self, conf_file: str):
```

Comment:
Curious about where is this function called.

Reply:
The function will be called when we run` spark_rapids_validation_tool validation --conf_file=datavalid_conf.yml`, it is generated as a command by the python [fire](https://github.com/google/python-fire) module

Reply:
Thanks for the explanation.